build/config/BUILDCONFIG.gn

```diff
@@ -360,7 +361,9 @@ default_compiler_configs = [
   "//build/config/sanitizers:default_sanitizer_flags",
 ]
 
-default_compiler_configs += [ "//chrobalt/build/configs:chrobalt_config" ]
+if (chrobalt) {
+  default_compiler_configs += [ "//chrobalt/build/configs:chrobalt_config" ]
```

Comment:
Try to manually gn format those changed files. Again, presubmits don't work for the moment.

Reply:
i normally run gn format but i think many files the changes have not been saved or something and got pushed wrongly. i'll doa follow up cl with clenups, probably by running gn format